tests/pivots/pivot_measure/pivot_measure_display_model.test.ts

```diff
@@ -1607,11 +1607,11 @@ describe("Measure display", () => {
           display: measureDisplay,
         },
         {
-          fieldName: "Expected Revenue + 1000",
+          fieldName: "Expected Revenue + 10%",
```

Comment:
l'm lowering the value just because 7% + 1000 gives an insanely large percentage, and all values are (relatively) almost equal